x-pack/plugins/watcher/public/application/models/watch/json_watch.js

```diff
@@ -113,7 +114,7 @@ export class JsonWatch extends BaseWatch {
     return new JsonWatch(upstreamWatch);
   }
 
-  static defaultWatchJson = defaultWatchJson;
+  static defaultWatch = defaultWatch;
```

Comment:
Reviewers, I need to do a quick grep and see if this renaming will cause any problems.

Reply:
Looks like this isn't referenced anywhere in the codebase, so I removed it.